packages/components/src/Autocomplete/Autocomplete.tsx

```diff
@@ -16,11 +16,31 @@ interface AutocompleteProps {
    */
   readonly value: Option | undefined;
 
+  /**
+   * Hint text that goes above the value once the form is filled out.
+   */
+  readonly menuSize?: "medium";
```

Comment:
`"Base"` would be better here to follow the standard size for other Atlantis components.

Do we only accept one size here?

Reply:
yep only one size. Instead should we make this `fixedSize` or something?

Reply:
if there is only ever one size can we remove the option altogether?